abs.yaml

```diff
@@ -0,0 +1,30 @@
+upload_channels:
```

Comment:
All staging channels can be removed as they've already been merged (except for `pytorch-metric`)